openmrs/omod/src/main/java/org/openmrs/projectbuendia/webservices/rest/OrderResource.java

```diff
@@ -270,6 +270,7 @@ private void populateDefaultsForAllOrders(Order order) {
         // Will be null if `orderer` is null.
         User creator = Utils.getUserFromProvider(orderer);
         order.setEncounter(createEncounter(order.getPatient(), creator, new Date()));
+        order.setCreator(creator);
```

Comment:
In the previous PR this was necessary - it was a fix for the same problem as the `patient.setCreator(CREATOR)` issue I described below. Seeing as I eliminated the `CREATOR` variable in this class a while ago it probably shouldn't be necessary here.